cmd/helm/status.go

```diff
@@ -140,11 +138,9 @@ func (s statusPrinter) WriteTable(out io.Writer) error {
 		_, _ = fmt.Fprintf(out, "VERSION: %s\n", s.release.Chart.Metadata.Version)
 		_, _ = fmt.Fprintf(out, "APP_VERSION: %s\n", s.release.Chart.Metadata.AppVersion)
 	}
-	if s.showDescription {
-		_, _ = fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
-	}
+	_, _ = fmt.Fprintf(out, "DESCRIPTION: %s\n", s.release.Info.Description)
```

Comment:
This change is breaking some tests and they don't expect the description to be present.

Reply:
I accidentally started this branch from #13444, I'll cherry-pick the commits to a new branch and re-open the PR

Reply:
Reopened as #13460